.gitignore

```diff
@@ -6,3 +6,6 @@ output/
 /bin/
 /.classpath
 /.project
+src/java/boa/datagen/misc
```

Comment:
why is a directory in src/ being ignored?

Reply:
This is to avoid accidentally committing files used by me only.
It looks stranger, so I will remove it.